common/lib/xmodule/xmodule/modulestore/tests/test_assetstore.py

```diff
@@ -573,79 +573,6 @@ def test_delete_all_different_type(self, storebuilder):
 
             assert len(store.get_all_asset_metadata(course.id, 'different')) == 1
 
-    @ddt.data(*MODULESTORE_SETUPS)
-    def test_get_all_assets_with_paging(self, storebuilder):
-        """
-        Save multiple metadata in each store and retrieve it singularly, as all assets, and after deleting all.
-        """
-        with storebuilder.build() as (__, store):
-            course1 = CourseFactory.create(modulestore=store)
-            course2 = CourseFactory.create(modulestore=store)
-            self.setup_assets(course1.id, course2.id, store)
-
-            expected_sorts_by_2 = (
```

Comment:
Nit: As the writer of this test, I still feel some ownership of it. How about, instead of deleting the whole test, delete `expected_sorts_by_2` and the `for` loop below? But leave the edge cases below it, which haven't failed (yet) and will still provide some code/test coverage?